app/helpers/ExerciseConfig/Pipeline/Box/Params/Priorities.php

```diff
@@ -12,4 +12,14 @@ class Priorities
     public static $EVALUATION = 80;
     public static $DEFAULT = 42;
     public static $DUMP_RESULTS = 1;
+
+    public static function defaultPriorityOfType(TaskType $type)
```

Comment:
Is this used somewhere? I cannot find any usages...

Reply:
No, I used it and then decided to take a different approach, but I think we can keep this code.